about.html

```diff
@@ -0,0 +1,150 @@
+
+          <p class="contact_us">
+            Please contact us now per Email to get started if you are ready join hundreds of entreprenuers for this edition of "MEET-UP TO START UP"
+          </p>
+          <a href="mailto:info@decagoninstitute.com" class="email_add">info@decagoninstitute.com</a>
+
+      </div>
+     </div>
+  </section>
+
+    <section class="about_logo_section">
+    <h2 class="about_logo_title">MEET-UP TO START UP</h2>
+    <div class="about_red_line"></div>
+      <p class="about_logo_text">
+        MEET-UP TO START UP is an idea geared to create millions of Startup companies 
+        all over the world having also been nourished by numerous incubators and have
+        investors ready to fund the next great venture.Although, there are numerous companies in that started small, they are making it big today,  so that startup ideas still have a fair chance if you will take a step.
+      </p>
+
+    <img class="about_logo" src="capstone-image/new logo.png" alt="startup logo">
+</section>
+
+<section class="previous_meetings">
+
+  <div class="previous_container">
+    <h2>See past MEET-UP TO START UP conferences</h2>
+    <div class="previous_line"></div>
+    <p>Take a look at the previous summits which was held in Johannesburg and CapeTown, South Africa</p>
+  </div>
+
+  <div class="previous_container2">
+    <div class="JBurg">
+      <h3 class="Jburg_date">2019</h3>
+      <p class="JBurg_text">Johannesburg 2019</p>
+    </div>
+
+    <div class="CTown">
+      <h3 class="CTown_date">2022</h3>
+      <p class="CTown_text">CapeTown 2021</p>
+    </div>
+  </div>
+
+</section>
+
+<section class="fourth_section">
+    <div class="sponsor_title">
+      <h2 class="sponsor_heading">Partners</h2>
+      <div class="red_line3 red_line4"></div>
+    </div>
+    <div class="sponsors_icon">
+      <img class="sponsors_logo" src="capstone-image/amazon.svg" alt="Amazon">
+      <img class="sponsors_logo" src="capstone-image/dropbox.svg" alt="Dropbox">
+      <img class="sponsors_logo" src="capstone-image/slack.svg" alt="Slack">
+      <img class="sponsors_logo" src="capstone-image/shopify.svg" alt="Shopify">
+      <img class="sponsors_logo" src="capstone-image/open-table.svg" alt="Open_table">
+      <img class="sponsors_logo" src="capstone-image/hubspot.svg" alt="Hubspot">
+  
+    </div>
+  </section>
+
+   <footer class="footer_section">
+    <img class="footer_image" src="capstone-image/new logo.png" alt="startup">
+    <div class="reserved_rights_container">
+      <h4 class="footer_rights">2023 DECAGON Project. Supported by Rapheal Okonta.</h4>
+      <p class="reserved_rights">All rights reserved.</p>
+      <p class="closing_text">THE DECAGON GROUP ACCELERATING INNOVATIONS AND STARTUPS FOR THE NEXT GENERATION.</p>
+    </div>
+  </footer>
```

Comment:
- [ ] Kindly consider the footer, it supposes be have a dark background as the design guideline.